baybe/recommenders/pure/bayesian/base.py

```diff
@@ -57,10 +57,10 @@ def _get_acquisition_function_cls(
             "PM": PosteriorMean,
             "PI": ProbabilityOfImprovement,
             "EI": ExpectedImprovement,
-            "UCB": partial(UpperConfidenceBound, beta=1.0),
+            "UCB": partial(UpperConfidenceBound, beta=0.1),
```

Comment:
context? is 1 not the standard choice?

Reply:
I think there is no "standard" choice, but 0.1 is used right on the start page of botorch and I had the impression that for Hartmann it indeed produced better results — which is of course just one random example... Happy to change it back. Perhaps you are right and we should only do what is really necessary in this quickfix...

Reply:
i would rather not touch it in this PR, if you need to chagne it then use 0.2 as it appears often in the botorch docs as default
this change ideally is rather temporary anyway as I would like to work on the acqf revamp including the configurability of this rather soon

Reply:
no prop, dropped the commit with the change